src/hypertrace/agent/init/__init__.py

```diff
@@ -57,49 +54,63 @@ def __init__(self, agent_config: AgentConfig, init_console_only: bool = False):
             self._postgresql_instrumentor_wrapper = None
             self._requests_instrumentor_wrapper = None
             self._aiohttp_client_instrumentor_wrapper = None
+
         except Exception as err:  # pylint: disable=W0703
             logger.error('Failed to initialize tracer: exception=%s, stacktrace=%s',
                          err,
                          traceback.format_exc())
             raise sys.exc_info()[0]
 
+    def init_trace_provider(self) -> None:
+        self._tracer_provider = TracerProvider(
+            resource=Resource.create({
+                "service.name": self._config.agent_config.service_name,
+                "service.instance.id": os.getpid(),
+            })
+        )
+
+        trace.set_tracer_provider(self._tracer_provider)
+
+    def init_propagation(self) -> None:
+        print('------')
+        print(type(self._config.agent_config.propagation_formats))
```

Comment:
Here I tried to understand the value for it with no success yet.

Reply:
Fixed.